artemis-server/src/main/java/org/apache/activemq/artemis/core/server/ServiceRegistry.java

```diff
@@ -32,32 +32,12 @@
 {
```

Comment:
Based on the change in use case (which I don't see in the ticket) should this be `ImmutableServiceRegistry` instead?